lr-cli/cortx_setup/commands/resource/show.py

```diff
@@ -39,7 +43,7 @@ def resource_filter_status(self, resource_state, resource_map):
         else:
             return resource_map
 
-    def parse_resource_file(self, resource_map_path: str):
+    def parse_file(self, resource_map_path: str):
```

Comment:
Change name from `resource_map_path` to: `file_map_path`

Reply:
Done